frontend/cypress/e2e/links.cy.js

```diff
@@ -0,0 +1,52 @@
+/// <reference types="Cypress" />
+
+context('Links on site', () => {
+  const routes = [
+    "/",
+    "explore/Human-GEM/gem-browser",
+    "/explore/Human-GEM/gem-browser/metabolite/MAM01282c",
+    "/explore/Human-GEM/map-viewer/golgi_apparatus?dim=2d&panel=0&sel=&search=&coords=-4012.67,-15817.92,0.2,0,0,500&dataTypes=None&dataSources=None&dataSets=None",
+    "/explore/Human-GEM/interaction-partners",
+    "/explore/Human-GEM/interaction-partners/MAM01632e",
+    "/gems/repository",
+    //"/gems/repository/Fruitfly-GEM",
```

Comment:
Is there any special reason this is commented out?

Reply:
Thank you for pointing that out. I should have reported that as a bug but forgot about it while looking into other problematic pages. I think there is a problem when navigating to that URL directly without passing `/gems/repository`. I think it should be reported as a bug separately since it seems to be a problem in `main` already.

Reply:
+1 for reporting that bug separately (with a note on amending the cypress test).